net/message/message.go

```diff
@@ -227,7 +227,8 @@ func (m *NotifyETHLocked) Type() Type {
 
 // NotifyXMRLock is sent by XMRMaker to XMRTaker after locking his XMR.
 type NotifyXMRLock struct {
-	Address string
+	Address string // address the monero was sent to
```

Comment:
note to self: I think this can *probably* be removed since the address can be derived from the two parties's public keys.